### ISSUE TITLE
Differentiate between DMs and regular statuses on the home page

### ISSUE THREAD

**Issue by mack-carter** [mastodon/mastodon#15914]
### Pitch

Either:

- Add a tag on DM statuses that shows in the home status feed (possibly something that changes the color?)
- Don't show direct messages in the home feed so they stay isolated from regular statuses in the UX

### Motivation

I just used the DM feature for the first time and realized that it functions as a status. Right now that means it shows up in the home feed without any indicator that it's a private direct message and not a public status. With the convenient reply button it could be easy to forget that a specific status is private or public when writing a reply. People tend to communicate differently when they are in private conversations as opposed to public statuses. Keeping DMs isolated in the inbox or adding a tag/visual cue in the home feed would help people differentiate the context in which they are communicating.

**Comment by ClearlyClaire** [comment]
There is currently an icon on the side of the timestamp showing that DMs are indeed DMs:
![image](https://user-images.githubusercontent.com/384364/111435729-6be7c300-8701-11eb-8d06-c3f5cd41d508.png)

I do agree that it's not enough of a visual clue, though. Your first suggestion is covered by #15594 and your second one by #15596 but @Gargron has yet to comment on any of them.

**Comment by mack-carter** [comment]
Good callout, thanks @Claire

Could just change the color on that icon in the UI, should be a simple fix



________________________________

**Comment by ClearlyClaire** [comment]
That is also a possibility, and some forks have an option for that. In the meantime you could add something like Stylus to add a userstyle like:

```css
.status__visibility-icon.fa-envelope {
    color: #d32f2f;
}
```

**Comment by mack-carter** [comment]
Oooh very interesting! I’m avoiding chrome atm, any favorite forks you could suggest?



________________________________

**Comment by ClearlyClaire** [comment]
Stylus is also available for Firefox.

I personally maintain a Mastodon fork called glitch-soc, it does feature high-color visibility icons as an option.

**Comment by mack-carter** [comment]
Awesome, I’ll check it out. Looking to get my own pub server up and running once I put all my cluster bits together, playing with some RPis



________________________________

**Comment by shleeable** [comment]
I deeply hate having DMs in the timeline. the lack of colour is also weird.

**Comment by trwnh** [comment]
Duplicate of #3819

**Comment by ClearlyClaire** [comment]
> #3819

I don't think so, but it has been addressed anyway in #24248
